app-drawer/app-drawer.html

```diff
@@ -260,70 +258,139 @@
 
       _scrimTapHandler: function() {
         if (!this.persistent) {
-          this.opened = false;
+          this.debounce('_scrimTapHandler', function() {
+            this.opened = false;
+          }, 1);
         }
       },
 
       _trackHandler: function(event) {
         if (!this.persistent) {
+          // Disable user selection.
+          event.preventDefault();
```

Comment:
This seems to work at least when dragging the drawer horizontally on desktop (can still select when dragging vertically). I don't want to disable user selection altogether, but I found that the selection is causing a repaint when I close the drawer on desktop.

I'll update the comment above to specify that this is for desktop.